modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/prepare/ddl/AlterTableAddCommand.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.query.calcite.prepare.ddl;
+
+import java.util.List;
+import org.apache.ignite.schema.definition.ColumnDefinition;
+
+/**
+ * ALTER TABLE ... ADD COLUMN statement.
+ */
+@SuppressWarnings("AssignmentOrReturnOfFieldWithMutableType")
+public class AlterTableAddCommand extends AbstractTableDdlCommand {
+    /** Quietly ignore this command if column already exists. */
+    private boolean ifColumnNotExists;
+
+    /** Columns. */
+    private List<ColumnDefinition> cols;
+
+    public List<ColumnDefinition> columns() {
+        return cols;
+    }
+
+    public void columns(List<ColumnDefinition> cols) {
+        this.cols = cols;
+    }
+
+    public boolean ifColumnNotExists() {
+        return ifColumnNotExists;
+    }
+
+    public void ifColumnNotExists(boolean ifColumnNotExists) {
```

Comment:
I've never saw the syntax when "if exists" was applies to the column.
Is it ever useful?

Reply:
[useless](https://ignite.apache.org/docs/latest/sql-reference/ddl#alter-table) we made it like in h2.

Reply:
Postgres support such syntax: https://www.postgresql.org/docs/9.6/sql-altertable.html

Reply:
I think H2 syntax look ambigous, because 
1. it is not clear if the flag relates to a first (or single column) or to all the columns?
2. if one of column exists, will the other columns be added? 

Postgres API looks ok, ADD COLUMN allows only one column as an argument.